### PR TITLE
feat(config): support update the region field in customizing OSS endp…

### PR DESCRIPTION
**Pull request by gaetanars**
Same as huaweicloud/terraform-provider-huaweicloud#2625 but for OSS

```
❯ go test -v ./flexibleengine -run TestCheckOssEndpoint
=== RUN   TestCheckOssEndpoint
--- PASS: TestCheckOssEndpoint (0.00s)
PASS
ok      github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/flexibleengine 0.030s
```

### CODE REVIEW

File: flexibleengine/config_test.go [hunk 1]
```diff
@@ -534,3 +535,27 @@ func TestAccServiceEndpoints_Others(t *testing.T) {
...
+		Cloud:  "prod-cloud-ocb.orange-business.com",
+	}
+
+	// without customizing OSS endpoint in Config
+	expected := "https://oss.eu-west-1.prod-cloud-ocb.orange-business.com/"
+	th.AssertEquals(t, expected, getOssEndpoint(cfg, "eu-west-1"))
+
+	// with customizing OSS endpoint in Config
+	cfg.Endpoints = map[string]string{
+		"obs": "https://oss.eu-west-0.prod-cloud-ocb.orange-business.com/",
```
Comment (ShiChangkuo) [inline flexibleengine/config_test.go:551]: The key of `getOssEndpoint` is **oss**, but we add **obs** in customizing endpoints.

It looks strange that for Object Storage Service user should add two keys if they want to customize the endpoint.
- **oss** for s3 resources
- **obs** for obs resources

I will check the short name of Object Storage Service and combine them into one.

Comment (gaetanars) [reply flexibleengine/config_test.go:551]: You're right, I see that this error existing before, and oss key doesn't exists in customizing endpoints.

Comment (ShiChangkuo) [reply flexibleengine/config_test.go:551]: **oss** used to be the short name of Object Storage Service, now the it's called **OBS**
but the endpoint is still using `oss.eu-west-0.prod-cloud-ocb.orange-business.com`

so, we should get the **obs** key in `getOssEndpoint` function.

Comment (gaetanars) [reply flexibleengine/config_test.go:551]: This is correct now

**Pull request merged**